docs/features/software-catalog/well-known-relations.md

```diff
@@ -0,0 +1,80 @@
+---
+id: well-known-relations
+title: Well-known Relations between Catalog Entities
+sidebar_label: Well-known Relations
+description: Documentation that lists a number of well known Relations, that
+have defined semantics. They can be attached to catalog entities and consumed
+by plugins as needed.
+---
+
+This section lists a number of well known
+[entity relation types](descriptor-format.md#common-to-all-kinds-relations),
+that have defined semantics. They can be attached to catalog entities and
+consumed by plugins as needed.
+
+## Relations
+
+This is a (non-exhaustive) list of relations that are known to be in active use.
+
+Each relation has a _source_ (implicitly: the entity that holds the relation), a
+_target_ (the entity to which the source has a relation), and a _type_ that
+tells what relation the source has with the target. The relation is directional;
+there are commonly pairs of relation types and the entity at the other end will
+have the opposite relation in the opposite direction (e.g. when querying for
+`A`, you will see `A.ownedBy.B`, and when querying `B`, you will see
+`B.ownerOf.A`).
```

Comment:
Perhaps a quick mention of adding new relations to `packages/catalog-model/src/kinds/relations.ts` and following guidelines in the comments there?

Reply:
done!